everyone-on-support.md

```diff
@@ -0,0 +1,33 @@
+# Introduction
+
+Everyone on Support (EOS) is a regular activity wherein all full-time members are called upon to do a Support shift once every month (this includes customer and tech support). Trialists and part-time team members are excluded from this.
+
+EOS is not designed to make everyone Support or to be pulled from regular work to do support. The main reason is that a new set of eyes can help find issues and opportunities in current processes. A developer might see something that is not obvious to a non-dev. A marketer might see a great opportunity that was dismissed till then.
+
+In the end, we are working for our customers, and having everyone involved in the conversation will help improve their experience and make them happy. And happy customers are happy to spend money with us.
+
+# Mechanics
+
+### I. Instructions
+
+* We use [GrooveHQ](https://niteoweb.groovehq.com/) for handling support emails. For EOS, we'll have one account which you'll need to personalize when you're on duty. Login credentials are on 1Password. Go to Settings and update the Name field.
+* Everyone has Staff access to the EBN app.
+* GrooveHQ integrates with Slack so we get a live stream of emails and responses. You can optionally join #support-emails channel.
+* You'll need to brush up on proper support decorum. See [Handling Support Requests](https://github.com/niteoweb/handbook/blob/master/support.md).
+* All support documents can be found in  [Intra EBN folder](https://intra.niteoweb.com/projects/easy-blog-networks/support-documents?_authenticator=dfadc77722eb429e0c160346cceba5b05d39702e). Make sure you go through them regularly and suggest improvements where needed.
```

Comment:
@marbeann: did you do Actions -> Click for Permalink -> right click -> copy address?

Left-clicking does not work, yet, there's a bug with Javascript.